utils/db.ts

```diff
@@ -378,3 +378,23 @@ export async function getUsersByIds(ids: string[]) {
   const res = await kv.getMany<User[]>(keys);
   return res.map((entry) => entry.value!);
 }
+
+export async function incrementVisitsPerDay(date: Date) {
+  // convert to universal timezone (UTC)
+  const visitsKey = [
+    "visits",
+    `${date.getUTCFullYear()}-${date.getUTCMonth()}-${date.getUTCDate()}`,
+  ];
+  await kv.atomic()
+    .sum(visitsKey, 1n)
+    .commit();
+}
+
+export async function getVisitsPerDay(date: Date): Promise<bigint> {
+  const visitsKey = [
+    "visits",
+    `${date.getUTCFullYear()}-${date.getUTCMonth()}-${date.getUTCDate()}`,
+  ];
+
+  return (await kv.get(visitsKey)).value as bigint;
+}
```

Comment:
```suggestion
export async function getVisitsPerDay(date: Date) {
  const res = await kv.get<bigint>([
    "visits",
    `${date.getUTCFullYear()}-${date.getUTCMonth()}-${date.getUTCDate()}`,
  ]);
  return res.value;
}
```
To make it consistent with other DB get functions.